### PR TITLE
Clarify the "Test Bugsnag" form action

### PR DESCRIPTION
**Pull request by imjoehaines**
## Goal

The current wording of the "Test Bugsnag" action is somewhat unclear

The intention is to test that we can communicate with Bugsnag using the given API Key, not to be a comprehensive test of the Bugsnag client, but that isn't currently made clear

This PR separates the test action from the main form and adds some clarifying text to make clear what the form is intended for

**Before**
before

**After**
Screenshot 2020-04-23 at 16 50 21

## Tests

This was tested manually be ensuring the test button still notifies Bugsnag with a test message

### Outstanding Questions

The current wording reads a bit awkwardly to me but I'm struggling to come up with anything better — any suggestions would be greatly appreciated!

### Linked issues

Related to #43 
StyleCI has been fixed in #45 so it will fail in this branch until that PR is merged

## Review

The changeset is much easier to review by ignoring whitespace changes!

For the submitter, initial self-review:

- [x] Commented on code changes inline explain the reasoning behind the approach
- [x] Reviewed the test cases added for completeness and possible points for discussion
- [x] A changelog entry was added for the goal of this pull request
- [x] Check the scope of the changeset - is everything in the diff required for the pull request?
- This pull request is ready for:
  - [ ] Initial review of the intended approach, not yet feature complete
  - [ ] Structural review of the classes, functions, and properties modified
  - [x] Final review

For the pull request reviewer(s), this changeset has been reviewed for:

- [ ] Consistency across platforms for structures or concepts added or modified
- [ ] Consistency between the changeset and the goal stated above
- [ ] Internal consistency with the rest of the library - is there any overlap between existing interfaces and any which have been added?
- [ ] Usage friction - is the proposed change in usage cumbersome or complicated?
- [ ] Performance and complexity - are there any cases of unexpected O(n^3) when iterating, recursing, flat mapping, etc?
- [ ] Concurrency concerns - if components are accessed asynchronously, what issues will arise
- [ ] Thoroughness of added tests and any missing edge cases
- [ ] Idiomatic use of the language

### CODE REVIEW

File: views/settings.php [hunk 1]
```diff
@@ -1,100 +1,107 @@
...
 
+<?php if (!empty($this->apiKey)): ?>
+    <div>
+        <h2>Test your connection to Bugsnag</h2>
+
+        <p>Use this button to test if your API key is able to communicate with Bugsnag.</p>
+
+        <p>This will send a message that can be viewed in <a href="https://app.bugsnag.com">your Bugsnag Dashboard</a>.</p>
+
+        <p>This specifically tests your API key and field filters. Any other settings you have customised will not be used in order to ensure the settings on this page are correct.</p>
```
Comment (tomlongridge) [inline views/settings.php:95]: I think we should make this a little simpler:

```suggestion
        <p>Use this button to send a test event that can be viewed in <a href="https://app.bugsnag.com">your Bugsnag Dashboard</a>.</p>

        <p>Note - any <a href="https://docs.bugsnag.com/platforms/php/wordpress/configuration-options/">configuration options</a> applied in code will not be applied to this test event.</p>
```

Comment (imjoehaines) [reply views/settings.php:95]: Yeah this is way better
